package.json

```diff
@@ -2,54 +2,52 @@
   "name": "ember-cli-deploy-newrelic",
   "version": "0.0.0",
   "description": "The default blueprint for ember-cli addons.",
+  "keywords": [
+    "ember-addon",
+    "ember-cli-deploy-plugin"
+  ],
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
+  "repository": "",
   "scripts": {
     "build": "ember build",
     "start": "ember server",
     "test": "ember try:each"
   },
-  "repository": "",
-  "engines": {
-    "node": ">= 0.10.0"
+  "dependencies": {
+    "ember-cli-babel": "^6.5.1",
+    "ember-cli-deploy-plugin": "^0.2.9",
+    "git-rev-sync": "^1.9.1",
+    "request": "^2.81.0",
+    "request-promise": "^4.2.1"
   },
-  "author": "",
-  "license": "MIT",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.4.2",
-    "ember-ajax": "^2.0.1",
-    "ember-cli-app-version": "^1.0.0",
-    "ember-cli-dependency-checker": "^1.2.0",
-    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
-    "ember-cli-htmlbars": "^1.0.3",
-    "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-jshint": "^1.0.0",
-    "ember-cli-qunit": "^2.0.0",
-    "ember-cli-release": "^0.2.9",
-    "ember-cli-sri": "^2.1.0",
-    "ember-cli-test-loader": "^1.1.0",
+    "broccoli-asset-rev": "^2.5.0",
+    "ember-ajax": "^3.0.0",
+    "ember-cli": "2.13.3",
+    "ember-cli-dependency-checker": "^1.3.0",
+    "ember-cli-eslint": "^3.0.0",
+    "ember-cli-htmlbars": "^1.1.1",
+    "ember-cli-htmlbars-inline-precompile": "^0.4.0",
+    "ember-cli-inject-live-reload": "^1.4.1",
+    "ember-cli-qunit": "^4.0.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-shims": "^1.1.0",
     "ember-cli-uglify": "^1.2.0",
-    "ember-cli": "2.7.0",
-    "ember-data": "^2.7.0",
-    "ember-disable-prototype-extensions": "^1.1.0",
-    "ember-export-application-global": "^1.0.5",
-    "ember-load-initializers": "^0.5.1",
-    "ember-resolver": "^2.0.3",
-    "ember-welcome-page": "^1.0.1",
-    "loader.js": "^4.0.1"
+    "ember-disable-prototype-extensions": "^1.1.2",
+    "ember-export-application-global": "^2.0.0",
+    "ember-load-initializers": "^1.0.0",
+    "ember-resolver": "^4.2.1",
+    "ember-source": "^2.13.3",
+    "ember-welcome-page": "^3.1.1",
```

Comment:
do you need welcome-page? is it being used by the dummy app?

Reply:
this comes with default blueprint, there is no harm in it being in devDeps